tools/wheel_resolver/__init__.py

```diff
@@ -85,6 +95,10 @@ def main(
             click.echo(u)
             return
 
+    # We're currently hardcoding PyPI but we should consider allowing other
+    # repositories
+    locator = distlib.locators.SimpleScrapingLocator(url="https://pypi.org/simple")
```

Comment:
This is where we're setting the locator to use sys_tags instead of the tags that distlib generates internally, which are less complete.